config/HardwareLayout.json

```diff
@@ -14,10 +14,10 @@
                     ],
                     "hardwareState": "AVAILABLE",
                     "hostname": "localhost",
-                    "port": 6667
+                    "port": 15088
                 }
             ],
-            "name": "Datacenter"
+            "name": "Data-Center"
```

Comment:
interesting that you are making this change too - I have discarded this change in the other PR, but yes let us make this change as we have certain other hardware layout files that have this change.